examples/httpserver/bin/basic_writer_client.dart

```diff
@@ -20,11 +21,11 @@ Future main() async {
   };
 
   var request = await new HttpClient()
-      .post(InternetAddress.LOOPBACK_IP_V4.host, 4049, '/file.txt');
-  request.headers.contentType = ContentType.JSON;
-  request.write(JSON.encode(jsonData));
-  HttpClientResponse response = await request.close();
-  await for (var contents in response.transform(UTF8.decoder)) {
+      .post(InternetAddress.LOOPBACK_IP_V4.host, 4049, 'file.txt'); /*1*/
+  request.headers.contentType = ContentType.JSON; /*2*/
+  request.write(JSON.encode(jsonData)); /*3*/
+  HttpClientResponse response = await request.close(); /*4*/
+  await for (var contents in response.transform(UTF8.decoder /*5*/)) {
```

Comment:
At my window width, I can't see anything to the right of the `))`.

Reply:
This one is a bit trickier to fix. I've reworked the code, PTAL.